ds-cv-inference/mqtt/mqtt.go

```diff
@@ -61,22 +61,25 @@ func NewMqttConnection(connectionString string) Connection {
 var commandTopicFunction MQTT.MessageHandler = func(client MQTT.Client, msg MQTT.Message) {
 
 	var edgeXMessage map[string]string
-	if err := json.Unmarshal(msg.Payload(), &edgeXMessage); err != nil {
```

Comment:
why we remove this? there is case-switch logic below still using this non-empty value...

Reply:
Ahh ok, I see where it is used down lower now. Please move it down closure to code that uses it.